libraries/botbuilder-ai/tests/TestData/LuisRecognizer/MultipleIntents_PrebuiltEntity.json

```diff
@@ -3,6 +3,9 @@
   "intents": {
```

Comment:
Note: This file was changed to allow [this test change](https://github.com/microsoft/botbuilder-js/pull/3690/files#diff-75b0b2325fef30a46cda63afcbfe8752b1a2038b60e98e213cac391e380a22bfR281) to pass. This file was called "MultipleIntents..." but only had a single one.